ta-server/EmailSender.ts

```diff
@@ -0,0 +1,7 @@
+export class EmailSender {
+  
+  enviarEmail(emailAdress: string, msg: string): boolean {
```

Comment:
emailAddress é o certo. pensando melhor precisamos de from, to, subject e message, 4 parâmetros. todos strings